queries.sql

```diff
@@ -78,3 +78,49 @@ SELECT species, AVG(escape_attempts) AS avg_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
+/* What animals belong to Melody Pond? */
+SELECT animals.name as animals_name
+    FROM animals
+    JOIN owners on animals.owner_id = owners.id
+    WHERE owners.full_name = 'Melody Pond';
+
+/* List of all animals that are pokemon (their type is Pokemon). */
+SELECT animals.name as animal_name, species.name as type
+    FROM animals
+    JOIN species on animals.species_id = species.id
+    WHERE species.name = 'Pokemon';
+
+/* List all owners and their animals, remember to include those that don't own any animal. */
+SELECT owners.full_name AS owner_name, animals.name AS animal_name
+    FROM owners
+    LEFT JOIN animals ON owners.id = animals.owner_idz
```

Comment:
[OPTIONAL BUT RECOMMENDED] Please correct the spelling of `owner_id` you accidently pass `z` after `owner_id` check my suggestion for more understanding :slightly_smiling_face: 

```suggestion
    LEFT JOIN animals ON owners.id = animals.owner_id
```